### ISSUE TITLE
Meteor satellites lie about station coverage and are incredibly underwhelming

### ISSUE THREAD

**Issue by DaydreamIQ** [tgstation/tgstation#71383]
Reporting client version: 514.1589

## Round ID:
194670

## Reproduction:
So this is a thing i've noticed for a while now, but in this particular shift the engineering team had setup a total of six meteor sats. Which apparently totalled to "1147%" coverage of the station. Shortly later, two meteor storms hit leading to an evac because the sats didn't help in the slightest. Either the station coverage value needs to be fixed, or they need some serious buffs

**Comment by Magic101lvl** [comment]
Based. Three meteor sats already give like 500%, even if setup in one same place, when actualy % of cover is one or less procents.

**Comment by DaydreamIQ** [comment]
> I dunno how to break this to you chief, but the game was rigged from the start
> 
> ![image](https://user-images.githubusercontent.com/28870487/202918268-43c0876b-baab-4da2-8f76-51fa147d07f8.png)
> 
> (the first meteor wave happens literally thirty seconds after the last shield was emagged)

Its still an issue that they only cover about a ten tile radius despite saying they cover ten thousand times more

**Comment by Mooshimi** [comment]
Incorrect coverage percentage displayed

**Comment by Inari-Whitebear** [comment]
It's not "incorrect" exactly

It's related to what you need for the goal I think?
Not "the whole station is covered by shields"

**Comment by davethwave** [comment]
Yep. Think the whole 100 percent thing is just for the station goal and should be viewed as a minimal requirement to pass centcom expectations. The astroid shields are supposed to be used to cover vital areas and don't have a ton of range by themselves sadly.
